diff --unified --recursive --text orig/Telegram/SourceFiles/data/components/sponsored_messages.cpp new/Telegram/SourceFiles/data/components/sponsored_messages.cpp
--- orig/Telegram/SourceFiles/data/components/sponsored_messages.cpp	2024-05-28 19:32:48.000000000 +0300
+++ new/Telegram/SourceFiles/data/components/sponsored_messages.cpp	2024-05-29 13:51:41.810417291 +0300
@@ -193,7 +193,7 @@
 }
 
 bool SponsoredMessages::canHaveFor(not_null<History*> history) const {
-	return history->peer->isChannel();
+	return false;
 }
 
 void SponsoredMessages::request(not_null<History*> history, Fn<void()> done) {
